src/backend/langflow/template/frontend_node/memories.py

```diff
@@ -2,11 +2,17 @@
 
 from langflow.template.field.base import TemplateField
 from langflow.template.frontend_node.base import FrontendNode
+from langflow.template.template.base import Template
 
 
 class MemoryFrontendNode(FrontendNode):
     #! Needs testing
     def add_extra_fields(self) -> None:
+        # chat history should have another way to add common field?
```

Comment:
I agree. With your PR we might need to consider a new category. This solution will do for now.